### PR TITLE
feat(home page): here i created the home paage

### LINKED ISSUE #41: Home Page
**Task Title**
Create the home page.

**Task description** 
Create the project home page and include inside it the following components: Hero Section, Recently Added grid(make one), How It Works section, and the Testimonial Carousel.

**Subtasks**
- [x] #55
- [x] #56

**Screenshot:**

Image

[Figma Link](https://www.figma.com/file/FYZIsFvg1rLu50UnriUN8Y/Student-Store-Project?type=design&node-id=2-11749&mode=design&t=i0O5DvjMg4XZDRs4-4)

### PR DESCRIPTION
**Pull request by samiraTbl**
Home Page #41
description: home page creation
screenshot:
image

### CODE REVIEW

File: src/app/[locale]/home/page.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import AddedGrid from "@/components/addedGrid/AddedGrid";
```
Comment (0m3ga13) [inline src/app/[locale]/home/page.js]: No need for this file

File: src/components/addedGrid/AddedGrid.jsx [hunk 2]
```diff
@@ -1,43 +1,44 @@
 "use client";
```
Comment (0m3ga13) [inline src/components/addedGrid/AddedGrid.jsx:1]: I guess it's preferable to use a map here instead

**Review by 0m3ga13 — CHANGES_REQUESTED** [review]
I made two reviews, also I guess the Product Card needs props and we need some data to be fetched in the home page (fetching the last 8 products from the firebase).

@medshk is working on the product cards to give them the props so we will keep the home page on Hold until then, please check the other reviews meanwhile.

File: src/components/addedGrid/AddedGrid.jsx (hunk 2 above)
Comment (samiraTbl) [reply src/components/addedGrid/AddedGrid.jsx:1]: Yes, i thought about it, but i kept to the data fetching

**Review by 0m3ga13 — APPROVED** [review]

**Review by Bolphunga — APPROVED** [review]

**Pull request merged**